engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql

```diff
@@ -18,489 +18,51 @@
 --;
 -- Schema upgrade from 4.14.0.0 to 4.15.0.0
```

Comment:
i don´t think this file should contain any changes for this PR
schema-41710to41800.sql should contain these changes.

Reply:
i moved it from 4.15 to 4.18